data/channels.csv

```diff
@@ -25685,12 +25700,14 @@ VatiLao.la,Vati Lao,,,,LA,,,c/LA,lao,entertainment;shop,FALSE,,,,,https://i.imgu
 Vavoom.rs,Vavoom,,,,RS,,,c/RS,srp,,FALSE,,,,,https://i.imgur.com/d9s1See.png
 VavTV.tr,Vav TV,,,,TR,,,c/TR,tur,,FALSE,,,,https://www.vavtv.com.tr/,https://i.imgur.com/jw0gB8L.png
 VayasemanitaSpain.es,Vaya semanita Spain,,,,ES,,,c/ES,spa,,FALSE,,,,https://pluto.tv/en/live-tv/vaya-semanita-es,https://i.imgur.com/0bCLNUG.png
+VBCNews.in,VBC News,,,,IN,,,c/IN,mal,news,FALSE,,,,https://vbcnewsthodupuzha.com/,https://i.imgur.com/tdIewPd.png

```

Comment:
```suggestion
VBCNews.in,VBC News,,,,IN,,,c/IN,mal,news,FALSE,,,,https://vbcnewsthodupuzha.com/,https://i.imgur.com/SvXCjha.png
```
Removed the transparent background around the logo.

Reply:
The original image already has a transparent background!